package.json

```diff
@@ -60,8 +60,9 @@
     "xmlhttprequest": "1.5.0"
   },
   "scripts": {
-    "lint": "eslint --quiet",
-    "lint-loud": "eslint static/js frontend_tests"
+    "lint": "eslint --quiet --cache",
+    "lint-fix": "eslint --quiet --fix",
```

Comment:
What's the effect of adding this line? Is there something that will try to call it?

Reply:
we can fix most common linting errors. Now that i think about we should document it?